Library/SharedFunctions.swift

```diff
@@ -764,3 +748,47 @@ public func attributedCurrency(withProject project: Project, total: Double) -> N
     superscriptAttributes: checkoutCurrencySuperscriptAttributes()
   )
 }
+
+private func estimatedMinMax(
+  from rewards: [Reward],
+  selectedShippingRule: ShippingRule
+) -> (Double, Double) {
+  var min: Double = 0
+  var max: Double = 0
+
+  rewards.forEach { reward in
+    guard reward.shipping.enabled, let shippingRules = reward.shippingRules else { return }
+
+    /// if the current reward's shipping is unrestricted then
+
+    var shippingRule: ShippingRule?
+
+    /// If the reward's shipping prefernce is Anywhere in the world, us it's first and only shipping rule.
+    /// Else use the rule that  matches the selected shipping rule (from the locations dropdown).
+    shippingRule = reward.shipping.preference == .unrestricted
+      ? shippingRules.first
+      : shippingRules
+      .first(where: { $0.location.country == selectedShippingRule.location.country })
+
+    guard let shipping = shippingRule else { return }
+
+    /// Verify there are estimated amounts greater than 0
```

Comment:
Can you confirm that min estimated shipping needs to be greater than zero? Ie that it's not possible for creators to ex say shipping is 0-1 dollars? I don't think it's very likely that a creator would charge shipping for something like a postcard or letter during pledge redemption, but I think it could be possible

Reply:
I can confirm that you cannot have a minimum shipping value of 0.

Reply:
Great, thanks!